autofs-5.1.3 - fix some man page problems

From: Ian Kent <raven@themaw.net>

Fix a few problems with the autofs man pages:
- autofs(8) missing "autofs -" in NAME string.
- punctuation marks in some SEE ALSO sections.
- change NAME string to include key word "autofs" in auto.master(5).

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG                      |    1 +
 man/auto.master.5.in           |    4 ++--
 man/autofs.5                   |    4 ++--
 man/autofs.8.in                |    8 ++++----
 man/autofs.conf.5.in           |    2 +-
 man/autofs_ldap_auth.conf.5.in |    2 +-
 man/automount.8                |    4 ++--
 7 files changed, 13 insertions(+), 12 deletions(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -254,6 +254,7 @@
 - set systemd KillMode to process.
 - fix mount.nfs blocks on first mount.
 - fix typos in autofs man pages.
+- fix some man page problems.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/man/auto.master.5.in
+++ autofs-5.0.7/man/auto.master.5.in
@@ -1,7 +1,7 @@
 .\" t
 .TH AUTO.MASTER 5 "11 Apr 2006"
 .SH NAME
-auto.master \- Master Map for automounter
+auto.master \- Master Map for automounter consulted by autofs
 .SH "DESCRIPTION"
 The
 .B auto.master
@@ -362,7 +362,7 @@ configuration will be used to locate the
 .BR autofs (5),
 .BR autofs (8),
 .BR autofs.conf (5),
-.BR autofs_ldap_auth.conf (5)
+.BR autofs_ldap_auth.conf (5).
 .SH AUTHOR
 This manual page was written by Christoph Lameter <chris@waterf.org>,
 for the Dean GNU/Linux system.  Edited by <hpa@transmeta.com> and
--- autofs-5.0.7.orig/man/autofs.5
+++ autofs-5.0.7/man/autofs.5
@@ -596,8 +596,8 @@ to the "local" subdirectory of the exter
 .BR auto.master (5),
 .BR autofs (8),
 .BR autofs.conf (5),
-.BR mount (8).
-.BR autofs_ldap_auth.conf (5)
+.BR mount (8),
+.BR autofs_ldap_auth.conf (5).
 .SH AUTHOR
 This manual page was written by Christoph Lameter <chris@waterf.org>,
 for the Debian GNU/Linux system.  Edited by H. Peter Avian
--- autofs-5.0.7.orig/man/autofs.8.in
+++ autofs-5.0.7/man/autofs.8.in
@@ -1,6 +1,6 @@
 .TH AUTOFS 8 "9 Sep 1997"
 .SH NAME
-Service control for the automounter
+autofs \- Service control for the automounter
 .SH SYNOPSIS
 If a SysV init script system is being used:
 .br
@@ -72,9 +72,9 @@ for more information.
 .BR automount (8),
 .BR autofs (5),
 .BR autofs.conf (5),
-.BR auto.master (5).
-.BR autofs_ldap_auth.conf (5)
-.BR systemctl(1)
+.BR auto.master (5),
+.BR autofs_ldap_auth.conf (5),
+.BR systemctl(1).
 .SH AUTHOR
 This manual page was written by Christoph Lameter <chris@waterf.org>,
 for the Debi GNU/Linux system.  Edited by H. Peter Anvin
--- autofs-5.0.7.orig/man/autofs.conf.5.in
+++ autofs-5.0.7/man/autofs.conf.5.in
@@ -513,6 +513,6 @@ map_type = file
 .SH "SEE ALSO"
 .BR automount (8),
 .BR auto.master (5),
-.BR autofs_ldap_auth.conf (5)
+.BR autofs_ldap_auth.conf (5).
 .SH AUTHOR
 This manual page was written by Ian Kent <raven@themaw.net>.
--- autofs-5.0.7.orig/man/autofs_ldap_auth.conf.5.in
+++ autofs-5.0.7/man/autofs_ldap_auth.conf.5.in
@@ -113,6 +113,6 @@ externally configured credential cache t
 By default, autofs will setup a memory based credential cache.
 .SH "SEE ALSO"
 .BR auto.master (5),
-.BR autofs.conf (5),
+.BR autofs.conf (5).
 .SH AUTHOR
 This manual page was written by Ian Kent <raven@themaw.net>.
--- autofs-5.0.7.orig/man/automount.8
+++ autofs-5.0.7/man/automount.8
@@ -178,8 +178,8 @@ constructed has been detached from the m
 .BR autofs (8),
 .BR autofs.conf (5),
 .BR auto.master (5),
-.BR mount (8).
-.BR autofs_ldap_auth.conf (5)
+.BR mount (8),
+.BR autofs_ldap_auth.conf (5).
 .SH BUGS
 Don't know, I've fixed everything I know about.
 
